opennlp-tools/src/test/java/opennlp/tools/tokenize/SimpleTokenizerTest.java

```diff
@@ -128,4 +128,18 @@ void testTokenizationOfStringWithWindowsNewLineTokens() {
     Assertions.assertArrayEquals(new String[] {"a", "\r", "\n", "\r", "\n", "b", "\r", "\n", "\r", "\n", "c"},
         tokenizer.tokenize("a\r\n\r\n b\r\n\r\n c"));
   }
+
+  /**
+   * Tests if it can tokenize a word containing a non-spacing character
+   * like Arabic Damma Unicode Character “◌ُ” (U+064F)
+   */
+  @Test
+  void testNonSpacingLetters() {
+    String text = "طُوّر";
```

Comment:
@demq Can we have a full sentence example here?

Reply:
I have just pushed an update with a full sentence.